amida-tech/qtc-web-ui
# QTC-158: Develop page

# Name of Feature 

## Description
### Related Tickets

- [QTC-158](https://jira.amida.com/browse/QTC-158)

### How Things Worked (or Didn't) Before This PR
Only had filler Develop page

### How Things Work Now
Develop page complete with input form, API call, and file download

## How to Test

- Run `npm install`
- Run `npm start`
- Visit `localhost:1776/develop` in browser
- Fill out form with a valid DBQ number and thresholds, then click Submit and verify that the correct mapping file downloads to your machine

## Readiness

1. [ ] This PR has full test coverage (If this PR fixes a bug, you _must_ add test cases respresentative of the bug).
2. [ ] This PR passes all automated tests.
3. [ ] This PR has no linting errors.
4. [ ] This PR has no hardcoded UI strings or other obvious issues.
5. [ ] In addition to testing my changes while running this code via `npm start`, I tested my changes while running this code on my machine via `npm run build-prod-start` (necessary because the latter command mimics production).
6. [ ] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.
7. [ ] This PR's required changes outside of the scope of this repository have been documented in this pull request.
